pages/linux/getenforce.md

```diff
@@ -0,0 +1,8 @@
+# getenforce
+
+> Get the current mode of SELinux (i.e. enforcing, permissive, or disabled).
```

Comment:
```suggestion
> Get the current mode of SELinux (e.g. enforcing, permissive, or disabled).
```

Reply:
ie is `that is`
eg is `for example`

Reply:
I don't think that "i.e." is wrong here (if the provided list of possible modes is exhaustive)

Reply:
`i.e.` is correct here - the 3 modes are exhaustive.